rl/memory.py

```diff
@@ -99,3 +98,37 @@ def append(self, observation, action, reward, terminal):
     @property
     def nb_entries(self):
         return len(self.observations)
+
+
+class EpisodeParameterMemory(object):
+    def __init__(self,limit):
+        self.limit = limit
+
+        self.params = RingBuffer(limit)
+        self.intermediate_rewards = RingBuffer(200) # max episode steps : TODO make dynamic
```

Comment:
What about the TODO? Is this still something you want to do? If a fixed size buffer is sufficient, please simply add a second parameter to `__init__`.